tests/Browser/SettingsPageTest.php

```diff
@@ -31,13 +31,15 @@ public function testAdminCanChangeSiteName()
   {
     $this->browse(function (Browser $browser) {
       $browser->loginAs(User::first())->visit('/admin/settings')
-        ->assertSourceHas('<title>Laravel</title>')
+        ->assertSourceHas('<title>CSU Booking Platform</title>')
```

Comment:
maybe you should verify this against the config name or the value from settings db table?

Reply:
We already have a test for the backend. This isolates the test to the front end.